frontend/app/.server/environment/session.ts

```diff
@@ -24,7 +24,7 @@ export const session = v.object({
   SESSION_COOKIE_PATH: v.optional(v.string(), defaults.SESSION_COOKIE_PATH),
   SESSION_COOKIE_SAMESITE: v.optional(v.picklist(['lax', 'strict', 'none']), defaults.SESSION_COOKIE_SAMESITE),
   SESSION_COOKIE_SECRET: v.optional(
-    v.pipe(v.string(), v.minLength(32), v.transform(Redacted.make)),
+    v.pipe(v.string(), v.minLength(32), v.transform(Redacted<string>)),
```

Comment:
Why the generic type isn't inferred?

```suggestion
    v.pipe(v.string(), v.minLength(32), v.transform(Redacted)),
```

Reply:
🤷🏻

Typescript can't seem to figure out that the incoming value is a string.

Reply:
I suspect that the incoming type is not strictly a string.. perhaps a subtype or extension of it?

Reply:
I tried with this and it work.

```ts
function Test<t>(t: t): t {
  return t;
}

...

AZUREAD_CLIENT_SECRET: v.optional(v.pipe(v.string(), v.transform(Test))),
```

Reply:
I removed the callable class stuff and replaced it with a static factory method (`Redact.make()`). This follows the same pattern as effect-ts' redact module.

Please re-review.